builder/builder.go

```diff
@@ -172,11 +175,13 @@ func (b *Builder) ScheduleBuild(kb KogitoBuilder) (*api.Build, error) {
 			},
 		},
 	}
-
-	build, err := builder.NewBuild(platform, kb.ImageName, kb.PodMiddleName).
-		WithKanikoCache(kb.Cache).
-		WithKanikoResources(kb.Resources).
-		WithKanikoAdditionalArgs(kb.AdditionalFlags).
+	infoBuild := builder.BuilderInfo{Platform: platform,
```

Comment:
instead replicate can't we have a common helper func for it?
the build, err := part seems to be equal too.

Reply:
added